### ISSUE TITLE
Alpine 3.13 base image vulnerable to CVE-2021-28831

### ISSUE THREAD

**Issue by hairyhenderson** [alpinelinux/docker-alpine#164]
There's a recent CVE in `busybox` ranked with a CVSS3 rank of 7.5 (HIGH): https://nvd.nist.gov/vuln/detail/CVE-2021-28831

The current base image ships with v1.32.1-r3, while the fix is available in [v1.32.1-r4](https://gitlab.alpinelinux.org/alpine/aports/-/issues/12566). Can we get images re-spun with this newer version?

(nb: in the meantime, the workaround is to `RUN apk upgrade --no-cache busybox`)

Here's the output of `trivy i alpine:3.13.3`:

```console
$ trivy i alpine:3.13.3
2021-03-31T09:21:16.567-0400    INFO    Detecting Alpine vulnerabilities...
2021-03-31T09:21:16.568-0400    INFO    Trivy skips scanning programming language libraries because no supported file was detected

alpine:3.13.3 (alpine 3.13.3)
=============================
Total: 2 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 2, CRITICAL: 0)

+------------+------------------+----------+-------------------+---------------+---------------------------------------+
|  LIBRARY   | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |                 TITLE                 |
+------------+------------------+----------+-------------------+---------------+---------------------------------------+
| busybox    | CVE-2021-28831   | HIGH     | 1.32.1-r3         | 1.32.1-r4     | busybox: invalid free or segmentation |
|            |                  |          |                   |               | fault via malformed gzip data         |
|            |                  |          |                   |               | -->avd.aquasec.com/nvd/cve-2021-28831 |
+------------+                  +          +                   +               +                                       +
| ssl_client |                  |          |                   |               |                                       |
|            |                  |          |                   |               |                                       |
|            |                  |          |                   |               |                                       |
+------------+------------------+----------+-------------------+---------------+---------------------------------------+
```

**Comment by lclc** [comment]
Looks like this was fixed here:
https://alpinelinux.org/posts/Alpine-3.13.4-released.html
and then in this commit:
https://github.com/alpinelinux/docker-alpine/commit/db57c96bfff7363dd9bccc56a0ce6e846261bbf8

**Comment by hairyhenderson** [comment]
Thanks @lclc - I'll close this as it's fixed by db57c96
